src/constants.ts

```diff
@@ -48,6 +48,7 @@ export const NETWORK_ERRORS = {
   ECONNRESET: ErrorCodes.connectionRefused,
   ENETUNREACH: ErrorCodes.connectionRefused,
   ENOTFOUND: ErrorCodes.dnsNotFound,
+  EADDRNOTAVAIL: ErrorCodes.dnsNotFound, // happens when DNS cannot resolve the IPv6
```

Comment:
unused?

Reply:
@ArtsiomCh this gets thrown from the needle request if IPv6 cannot be used.

Reply:
May be it my lack of TS knowledge, but I can't see usage of `EADDRNOTAVAIL` anywhere in the code...